src/ralph/models/edx/browser.py

```diff
@@ -25,9 +23,7 @@ class BaseBrowserEventModel(BaseEventModel):
 
     event_source: Literal["browser"]
     page: Union[AnyUrl, Path]
-    session: Union[
-        constr(regex=r'^[a-f0-9]{32}$'), Literal[""]
-    ]
+    session: Union[constr(regex=r"^[a-f0-9]{32}$"), Literal[""]]  # noqa: F722
```

Comment:
It seems flake-8 finds a false positive here https://github.com/PyCQA/pyflakes/issues/542
added # noqa: F722